source/features/unwrap-unnecessary-dropdowns.tsx

```diff
@@ -36,17 +33,47 @@ async function unwrapNotifications(): Promise<void | false> {
 	button.textContent = `Group by ${button.textContent.toLowerCase()}`;
 }
 
+function replaceRerunDropdown(menu: Element): void {
+	const triggerButton = $('focus-group > button', menu);
+
+	// We only need to unwrap the re-run jobs menu
+	if (triggerButton.textContent.trim() !== 'Re-run jobs') {
+		return;
+	}
+
+	const container = menu.parentElement!;
+	container.classList.add('d-flex', 'gap-2');
+
+	for (const button of $$('button.ActionListContent', menu)) {
+		button.className = 'Button--secondary Button--medium Button';
+		container.append(button.cloneNode(true));
+	}
+
```

Comment:
Since `$$()` might throw, let's add the classes later:

```suggestion
	for (const button of $$('button.ActionListContent', menu)) {
		button.className = 'Button--secondary Button--medium Button';
		container.append(button.cloneNode(true));
	}

	container.classList.add('d-flex', 'gap-2');
```